awx/api/serializers.py

```diff
@@ -5002,6 +5004,32 @@ class Meta:
         fields = read_only_fields
 
 
+class HostMetricSerializer(BaseSerializer):
+    show_capabilities = ['delete']
```

Comment:
I'd also prefer to not add user_capabilities with this line, given the basic nature of the permission rules.

Reply:
@AlanCoding can you be more specific here? We don't know the pitfalls of the permission rules. Is this needed to be able to delete via API?

Reply:
User capabilities are extra data from API dumped into "summary_fields" but the purpose is ultimately for the UI. The UI doesn't always need it, but if no UI is even planned then the need for the data won't exist.

Reply:
@AlanCoding the UI for this feature is currently in progress